src/osd/OSD.cc

```diff
@@ -5696,6 +5716,12 @@ bool remove_dir(
       cont = dstate->pause_clearing();
       handle.suspend_tp_timeout();
       waiter.wait();
+      if (cct->_conf->osd_remove_sleep) {
+        utime_t t;
```

Comment:
needs indentation

Reply:
I got rid of the tab characters so it would look right here.  I hate the inconsistency because we have a mix and GitHub doesn't render tabs right.  Sam always wanted tabs instead of just spaces.